configs/envoyproxy_io_proxy_http3_downstream.template.yaml

```diff
@@ -6,6 +6,51 @@ admin:
       address: 0.0.0.0
       port_value: 9901
 static_resources:
+  listeners:
```

Comment:
i think that this file is not a template - ie its pure envoy config (iiuc) - iirc having template in the filename will prevent it being validated